components/header.tsx

```diff
@@ -10,12 +10,13 @@ export default function Header() {
 
     useEffect(() => {
         // TODO: Move this outside of header, probably in _app.tsx (need to create an AuthProvider I think)
-        const publicRoutes = ['/', '/login', '/resetpassword'];
+        const publicRoutes = ['/', '/login', '/resetpassword', '/faq', '/r'];
```

Comment:
/faq is a not available route right?

Reply:
oppsie

Reply:
Resolve in https://github.com/nubpro/nakvaksin/pull/27/commits/7d7c464b4045a737f2794ec9f7f37dbed8aa6649